app/templates/events/event_list.html

```diff
@@ -96,12 +96,15 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
               {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
                 <td><a href="/event/{{event.id}}/edit" class="{{link_class}}">{{event.name}}</a></td>
             {% else %}
-              {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
-              <td><a href="/event/{{event.id}}/view" class="{{link_class}}">{{event.name}}</a></td>
+              <td><a href="/event/{{event.id}}/view" class="link-primary">{{event.name}}</a></td>
             {% endif %}
-            <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
-            <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-            <td class="text-dark">{{event.location}}</td>
+              {% if not user.isAdmin and event.isCanceled %}
+                <td colspan = 3 >This Event Has Been Canceled. Check your Email</td>
```

Comment:
Replace "Check your Email" with "Contact {{programManager}} if you have any questions." and in the case there is no program manager just say "Contact CELTS..."

Reply:
Also, we should not be capitalizing every word in the sentence.